pygame 2.1.2 (SDL 2.0.18, Python 3.8.12)
Hello from the pygame community. https://www.pygame.org/contribute.html
Current tracking mode: Default
Compute and apply returned calibration_status_success and collected at 6 points.
C:\Users\kessler_lab\Desktop\pupillometry-rf-back\titta\Tobii.py:1167: RuntimeWarning: Mean of empty slice
  np.nanmean(rms_l), np.nanmean(rms_r),
C:\Users\kessler_lab\Desktop\pupillometry-rf-back\titta\Tobii.py:1168: RuntimeWarning: Mean of empty slice
  np.nanmean(sd_l), np.nanmean(sd_r),
Saving calibration to file 1 for eye tracker with serial number X3120-030107707753.
calibration completed
1.7035 	WARNING 	Requested an unavailable screen number - using first available.
1.7042 	WARNING 	User requested fullscreen with size [1920 1080], but screen is actually [2560, 1440]. Using actual size
